app/configs.py

```diff
@@ -15,6 +16,9 @@ class OtaClientServerConfig:
     QUERYING_SUBECU_STATUS_TIMEOUT: float = 120  # 2mins
     LOOP_QUERYING_SUBECU_STATUS_INTERVAL: float = 10
 
+    # proxy server
+    OTA_PROXY_SERVER_ADDR: Tuple[str, int] = ("0.0.0.0", 8000)
```

Comment:
Any number is OK, number of 8000, aws greengrass seems using it.
```suggestion
    OTA_PROXY_SERVER_ADDR: Tuple[str, int] = ("0.0.0.0", 8082)
```